package com.javaex.exception.ex02;

import java.io.IOException;

public class ThrowExepApp {

	public static void main(String[] args) {
		
		
		
		ThrowExep exep = new ThrowExep();
		
		/*ThrowExep exep = new ThrowExep();
		try {
			exep.executeExcept();
		} catch (IOException e) {
			
			System.out.println(e);
			System.out.println("파일을 생성합니다.");
		}*/

	}

}
